setup.js

```diff
@@ -30,42 +30,42 @@ function setup_virtualenv(on_success, on_error) {
   )
 }
 
-function install_firebolt_sdk(python_dir, on_success, on_error) {
-  exec(path.join(python_dir, "pip") + " install firebolt-sdk",
-    function(error, stdout, stderr) {
+function install_python_dependencies(python_dir, on_success, on_error) {
+  exec(path.join(python_dir, "pip") + " install firebolt-sdk retry",
+    function (error, stdout, stderr) {
       error == null ? on_success(python_dir) : on_error(error.message);
     }
   )
 }
 
 function start_db(python_dir, on_success, on_error) {
-  exec(path.join(python_dir, 'python') + ' ' + resolve_local_file('scripts/start_database.py') + ' ' + core.getInput('db_suffix'),
-    { env: fb_env },
-    function(error, stdout, stderr) {
-      error == null ? on_success(stdout.trim('\n'), python_dir) : on_error(error.message);
-    });
+  const result = spawnSync(path.join(python_dir, 'python'),
```

Comment:
Any reason to replace `exec` with `spawnSync`?
If yes, please also change the `install_python_dependencies` function to be consistent

Reply:
And modify the cleanup.js script as well

Reply:
Yeah, it was flagged by the sonar as a security hotspot. https://github.com/firebolt-db/integration-testing-setup/runs/8595272076

The others are not vulnerable since they take no user input so I left them as is, but I can change them for consistency sake.

Reply:
Please do

Reply:
I've sunk too much time into trying to make it work. I'll continue work in a separate PR https://github.com/firebolt-db/integration-testing-setup/pull/11
For the sake of our nightlies and integration tests working again I'll merge this PR as is.